motion-of-no-confidence.md

```diff
@@ -0,0 +1,32 @@
+# Motion of no confidence
+
+The steering committee has [responsibilities][], and it's important
+that those responsibilities are enforceable.  For example, in the
+unlikely event that a committee member doesn't [attend 75% of the
```

Comment:
On Thu, Mar 12, 2015 at 12:51:20PM -0700, Katy Huff wrote:
> > @@ -0,0 +1,32 @@
> > +# Motion of no confidence
> > +
> > +The steering committee has [responsibilities][], and it's important
> > +that those responsibilities are enforceable.  For example, in the
> > +unlikely event that a committee member doesn't [attend 75% of the
> 
> Necessary to say unlikely? Let's stick to the facts in this document
> as much as possible.

Dropped in d056dc0 (motion-of-no-confidence.md: Drop 'unlikely',
2015-03-12).